ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -56,17 +57,19 @@ public String getVersion() {
         preOptimSensitivityAnalysisResult = SystematicSensitivityAnalysisService.runAnalysis(network, crac, computationManager);
         postOptimSensitivityAnalysisResult = preOptimSensitivityAnalysisResult;
         SystematicSensitivityAnalysisResult midOptimSensitivityAnalysisResult;
```

Comment:
can be renamed tempSensitivityAnalysisResult

Reply:
ok